src/screencast_keys/ops.py

```diff
@@ -1371,15 +1382,31 @@ def event_timer_remove(cls, context):
                 wm.event_timer_remove(cls.timers[key])
         cls.timers.clear()
 
-    def invoke(self, context, event):
-        cls = self.__class__
-        prefs = compat.get_user_preferences(context).addons[__package__].preferences
+    @classmethod
+    def start(cls, self, context, event, prefs, is_start):
```

Comment:
I think `is_start` argument is not good for understanding this classmethod.
Could you separate `start` method and `stop` method whether `is_start = True` or `is_start = False`?

Reply:
Good advice